### PR TITLE
PYIC-3417 Add new pages to support mitigation journeys

### PR DESCRIPTION
**Pull request by wilsond-gds**
## Proposed changes

Adding the pages as designed for journeys to allow users to continue proving their identity.

### What changed

New nunjucks and content pages. The Welsh `translation.json` file has had extra content removed that was preventing the JSON file from validating and additions to the page working predictably.

### Why did it change

To support new user journeys

### Issue tracking

- [PYIC-3417](https://govukverify.atlassian.net/browse/PYIC-3417)

[PYIC-3417]: https://govukverify.atlassian.net/browse/PYIC-3417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/locales/cy/translation.json [hunk 1]
```diff
@@ -163,6 +163,53 @@
...
+      "title": "Mae’n ddrwg gennym, bydd angen i chi brofi eich hunaniaeth mewn ffordd arall",
+      "header": "Mae’n ddrwg gennym, bydd angen i chi brofi eich hunaniaeth mewn ffordd arall",
+      "content": {
+        "paragraph1": "Mae hyn oherwydd nad ydych wedi ateb rhai o’r cwestiynau diogelwch yn gywir.",
+        "subHeading": "Beth allwch chi ei wneud",
+        "paragraph2": "Yn seiliedig ar beth rydych wedi’i ddweud wrthym, mae gennych chi lun ID y gallwch ei ddefnyddio i brofi pwy ydych chi.",
+        "paragraph3": "Gallwch wneud hyn gyda’r ap GOV.UK ID Check os oes gennych:\n",
+        "bullet1": "iPhone 7 neu uwch",
+        "bullet2": "ffôn Android (er enghraifft, Samsung neu Google Pixel)",
+        "paragraph4": "Gallwch hefyd brofi eich hunaniaeth mewn Swyddfa’r Post. Mae’r opsiwn hwn yn cymryd mwy o amser. Fel arfer, byddwch yn cael canlyniad o fewn diwrnod o fynd i Swyddfa’r Post.",
```
Comment (therealslimsladey) [inline src/locales/cy/translation.json]: Remove?

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -163,6 +163,52 @@
...
+        "paragraph3": "You can do this online with the GOV.UK ID Check app if you have:",
+        "bullet1": "an iPhone 7 or higher",
+        "bullet2": "an Android phone (for example, Samsung or Google Pixel)",
+        "paragraph4": "You can also prove your identity at a Post Office. This option takes longer. You’ll usually get an outcome within a day of going to the Post Office.",
+        "subHeading2": "What would you like to do?",
+        "formRadioButtons": {
+          "continueAppButtonText": "Prove your identity with the GOV.UK ID Check app",
+          "continueAppButtonTextHint": "You’ll be shown how to download and use the app.",
+          "continuePostOfficeButtonText": "Prove your identity at a Post Office",
+          "continuePostOfficeButtonTextHint": "You’ll be asked to enter details from your photo ID on GOV.UK first."
```
Comment (therealslimsladey) [inline src/locales/en/translation.json:182]: When F2F is added to the mitigation, do we really want to lose the 'Prove your identity another way' option?

**Review by therealslimsladey — CHANGES_REQUESTED** [review]
One minor issue and a question around the ucd design

File: src/locales/cy/translation.json (hunk 1 above)
Comment (wilsond-gds) [reply src/locales/cy/translation.json]: Oh that's highly annoying, removed

File: src/locales/en/translation.json (hunk 2 above)
Comment (wilsond-gds) [reply src/locales/en/translation.json:182]: From the UCD team: 

> The reasoning behind not giving "return to the RP" as an option here is a business decision that we want users to try the options available to them in our service first before sending them back the the RP. For a lot of RP's the alternative to proving their identity with us is similar to the F2F route or just a paper form, so F2F could potential cater to that anyway

File: src/locales/cy/translation.json (hunk 1 above)
Comment (therealslimsladey) [reply src/locales/cy/translation.json]: Are there commits to push? I can still see the extraneous paragraph4

**Review by AmritSidhu — DISMISSED** [review]

File: src/locales/cy/translation.json (hunk 1 above)
Comment (wilsond-gds) [reply src/locales/cy/translation.json]: Weird, no idea how that crept back in.

**Review by AmritSidhu — APPROVED** [review]

**Review by therealslimsladey — APPROVED** [review]
lgtm

**Pull request merged**